crates/bounded_planet_core/src/bin/main.rs

```diff
@@ -8,6 +8,7 @@ use bevy::{
 };
 use bounded_planet::{camera::*, land::*, land::TextureHeightmap};
 
+
```

Comment:
Are all the changes in this file necessary, since you have your own `egui_test.rs` above?

Reply:
Yeah, this was an accident to commit, I'll remove it later